modules/sql/src/main/resources/db/migration/V038__Add_Gpi.sql

```diff
@@ -27,7 +27,7 @@ CREATE TABLE e_gpi_filter (
   id          identifier  PRIMARY KEY,
   short_name  text        NOT NULL,
   long_name   text        NOT NULL,
-  band        identifier  REFERENCES e_magnitude_band,
+  band        identifier  NOT NULL REFERENCES e_magnitude_band,
```

Comment:
Do you need to change the code to use `EnumDef` here?

Reply:
Good question. is `MagnitudeBand` an enum? It would seem so

Reply:
Yes, I changed it, seems more consistent now